### ISSUE TITLE
Unreachable from other iOS device

### ISSUE THREAD

**Issue by Korcsog** [HomeACcessoryKid/ESP8266-HomeKit#30]
Hi!
The code works as charm, but if I change my device e.g. to an iPad, the accessory is unreachable (while from the iOS device I've paired from is still works). Any idea?

**Comment by HomeACcessoryKid** [comment]
Did you invite the user of the iPad to your home?
I did and it worked from both devices from thereon.

If it is the same user, it works from both.

Any special setup you have?

**Comment by Korcsog** [comment]
No, i'm not invited other users, because the iPad is mine too. So same iCloud account used...
After nearly one day, the ESP appears at the iPad, but sometimes i can't reach it from other devices. E.g. from the iPad works fine, but not reachable from my iPhone and vica versa.
Nothing special setup, just the demo code.

**Comment by HomeACcessoryKid** [comment]
In my experiments I did not dig into this aspect very deep, but I did notice that the account used to set up the device was the iPad of my wife. Then I invited myself. Now I could operate from the iPad, my iPhone and her iPhone. The last has been tested only a few times, but should represent your case. I did declare the iPad to be the home hub. Maybe that makes a difference.

Hope it starts working, this is Apples territory...
